### ISSUE TITLE
Results can't be reproduced

### ISSUE THREAD

**Issue by Anonymous-so** [layumi/University1652-Baseline#6]
Hi @layumi , thanks for releasing the codes.

When I ran the train.py file (using the resnet model), after initializing with the pretraining model parameters and training for 119 epochs, I ran the test.py file and only got the following results:
Recall@1:1.29 Recall@5:4.54 Recall@10:7.43 Recall@top1:7.92 AP:2.53

And when I ran the train.py file using the vgg mode, I got:
Recall@1:1.75 Recall@5:6.22 Recall@10:10.36 Recall@top1:11.16 AP:3.39

The hyper-parameters of the above results are set by default. To get the results in the paper, do I need to modify the hyper-parameters in the code?

I use pytorch 1.1.0 and V100

**Comment by layumi** [comment]
Hi @Anonymous-so 
1. The result is too low. It might be due to the environment or the dataset path.
- You may first try to download my trained model  and test them.  It is to check your test environment.

2. How you train the model? Which setting do you test? 
- Did you change the training setting? 
- Ground2Drone or Ground2Satellite is challenging, leading to low score.

**Comment by Anonymous-so** [comment]
Yes, as you say, after downloading the model you provided, the results are still incorrect.
I checked my environment.
python: 3.6.12
pytorch:1.6.0
numpy: 1.19.1
---------------------------------------
The testing setting is set default, i.e.,
which_epoch: last
name: usa_vgg_noshare_warm5_lr2
pool: avg
h: 256
w: 256
views: 2
fp16: True
use_dense: False
PCB: False
multi: False
stride: 2
ms: 1
use_NAS: False
-------------------------------------
The reported results:
Recall@1:1.89 Recall@5:5.90 Recall@10:9.26 Recall@top1:9.88 AP:3.35
--------------------------------------
It seems that the environments are incorrectly configured, but I checked the above and found that they all match the environments you suggest in [Prerequisites]. Are there any other environment options that need to be configured?
